src/pages/elements/color/tokens.mdx

```diff
@@ -8,6 +8,83 @@ tabs: ['Overview', 'Usage', 'Tokens', 'Code']
 
 import ColorTokenTable from 'components/ColorTokenTable';
 
-## Tokens by theme
+<PageDescription>
 
-<ColorTokenTable />
+Tokens are a method of applying color in a consistent, reusable, and scalable
+way. They are used in place of hard coded values, like hex codes.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Core Tokens</AnchorLink>
+<AnchorLink>Component Tokens</AnchorLink>
+<AnchorLink>AI Tokens</AnchorLink>
+
+</AnchorLinks>
+
+## Core Tokens
+
+Core tokens are global colors that are used across components. They are named
+and grouped by the common UI element that they are applied to.
+
+<AnchorLinks>
+
+<AnchorLink>Background</AnchorLink>
+<AnchorLink>Layer</AnchorLink>
+<AnchorLink>Layer accent</AnchorLink>
+<AnchorLink>Field</AnchorLink>
+<AnchorLink>Border</AnchorLink>
+<AnchorLink>Text</AnchorLink>
+<AnchorLink>Link</AnchorLink>
+<AnchorLink>Icon</AnchorLink>
+<AnchorLink>Support</AnchorLink>
+<AnchorLink>Focus</AnchorLink>
+<AnchorLink>Miscellaneous</AnchorLink>
+
+</AnchorLinks>
+
+<ColorTokenTable type="core" />
+
+## Component Tokens
+
+Some components have their own specific color tokens, known as component tokens.
+They represent the properties associated with a particular component. They are
+not global tokens like the core tokens and should never be used for anything
+other than their own component. See the
+
+<a href="https://github.com/carbon-design-system/carbon/blob/main/packages/styles/docs/sass.md#component-tokens">
+  developer documentation{' '}
+</a> for more information.
+
+<AnchorLinks>
+
+<AnchorLink>Button</AnchorLink>
+<AnchorLink>Tag</AnchorLink>
+<AnchorLink>Notification</AnchorLink>
+
+</AnchorLinks>
+
+<ColorTokenTable type="component" />
+
+## AI Tokens
+
+To accommodate for the new AI styles, we have introduced a new suite of color
+tokens that can be found within the main Carbon themes. These tokens should only
+be used when building custom AI components, variants, or instances in your UI.
+For more information on how to use the AI token, see the
+
```

Comment:
There is a weird line break here: 
<img width="835" alt="image" src="https://github.com/user-attachments/assets/888296aa-d8e9-4d34-beb8-b5aa1b4e6bf4">